platform_versioned_docs/version-24.2/compute-envs/azure-batch.mdx

```diff
@@ -250,17 +304,90 @@ Create a Batch Forge Azure Batch compute environment:
 See [Launch pipelines](../launch/launchpad.mdx) to start executing workflows in your Azure Batch compute environment.
 :::
 
-## Manual
+### Manual
 
-This section is for users with a pre-configured Azure Batch pool. This requires an existing Azure Batch account with an existing pool.
+You can configure Seqera Platform to use a pre-existing Azure Batch pool. This allows the use of more advanced Azure Batch features, such as custom VM images and private networking. See [Azure Batch security best practices][az-batch-best-practices] for more information.
 
 :::caution
 Your Seqera compute environment uses resources that you may be charged for in your Azure account. See [Cloud costs](../monitoring/cloud-costs.mdx) for guidelines to manage cloud resources effectively and prevent unexpected costs.
 :::
 
-Create a manual Seqera Azure Batch compute environment:
+#### Create a Nextflow-compatible Azure Batch pool
+
+If not described below, use the default settings:
+
+1. **Account**: You must have an existing Azure Batch account. Ideally, you should already have tested whether you can run an Azure Batch task within this account. Any type of account is compatible.
+1. **Quota**: You must check you have sufficient quota for the number of pools, jobs, and vCPUs per series. See [Azure Batch service quotas and limits][az-batch-quotas] for more information.
+1. On the Azure Batch page of the Azure Portal, select **Pools** and then **+ Add**.
+1. **Name**: Enter a Pool ID and Display Name. This ID will be used by Seqera and Nextflow.
+1. **Identity**: Select **User assigned** to use a managed identity for the pool. Select **Add** for the user-assigned managed identity and select the managed identity with the correct permissions to the Azure Storage and Batch accounts.
+1. **Operating System**: You can use any Linux-based image here, but it is recommended to use it with a Microsoft Azure Batch-provided image. Note that there are two generations of Azure Virtual Machine images, and certain VM series are only available in one generation. See [Azure Virtual Machine series][az-vm-gen] for more information. For default settings, select the following:
+    - **Publisher**: `microsoft-azure-batch`
```

Comment:
Just to confirm: Are these field entries or CLI? If they're in a UI field they don't need to be code format.

Reply:
That makes sense, thank you!